src/components/tvproviders/schedulesdirect.js

```diff
@@ -264,13 +264,15 @@ define(["jQuery", "loading", "emby-checkbox", "listViewStyle", "emby-input", "em
         self.init = function () {
             options = options || {};
 
-            if (options.showCancelButton) {
+            // Show cancel button by default
+            if (options.showCancelButton !== false) {
```

Comment:
Wtf, can you please explain?..

Reply:
Not 100% sure what's happening here but it feels like it would be better to rename this option to `hideCancelButton`. So that when the option unset/null the cancel button is still shown.

Reply:
This changes so that it is explicitly checking for a `false` value being set instead of hiding by default if the option is "falsey" (`undefined`, `null`, `0`, ...). This was the original behavior, but was broken in a previous commit as a suggested change in a PR. Changing options seems like it would be beyond the scope of a bugfix that should be backported.

https://github.com/jellyfin/jellyfin-web/commit/36fae11a38aeb1840a51496aa2334c8cce7c8c36#diff-e515aef7db17535709492337ab84cb24L269-L276

Reply:
At least add a //FIXME, I think Mark is right here in terms of renaming the option.